fastlane/Fastfile

```diff
@@ -778,7 +778,7 @@ platform :ios do
       res = file.find { |line| line =~ /^(?!\s*#)(?=.*\bgutenberg\b).*(\bcommit|tag\b){1}.+/ }
     end
 
-    UI.user_error!("Can't find any reference to Gutenberg!") unless res.length != 0
+    UI.user_error!("Can't find any reference to Gutenberg!") if res.empty?
```

Comment:
First Rubocop suggested to use `res.empty? == false` instead of `res.length != 0`, then to flip `unless res.empty? == false` to `if res.empty?`.

I'm spelling this out to make sure it holds logically, as I'm painfully aware that meddling with if statements can backfire 😅

Reply:
It reads that it's doing the same thing to me!